src/Previewer/vala-previewer/previewer.vala

```diff
@@ -20,14 +20,16 @@ namespace Workbench {
       });
     }
 
-    public void update_ui (string content, string target_id, string original_id) {
+    public void update_ui (string content, string target_id, string original_id = "") {
```

Comment:
Checking for undefined in the js side and vala side is twice, but I guess it doesn't hurt.